corehq/apps/notifications/views.py

```diff
@@ -97,7 +97,7 @@ def save_last_seen(self, in_data):
         }
 
     def get_domain(self):
-        domain_match_regex = r'(?<=a/)(.*?)(?=\s*/)'
+        domain_match_regex = r'(?<=/a/)([^\s/]{1,25}?)(?=\s*/)'
```

Comment:
This behavior is going to be slightly different, since instead of matching a minimum of 0 we're matching a minimum of 1 characters. Previously with a string like `/a//b/` we'd match `""` but now it will return `None`. This seems better to me, but it might be worth confirming this still fits the intended purpose. 

@minhaminha looks like you worked on this around 2 years ago, any chance you recall what this is for?

Reply:
Actually this should be since we're immediately checking it for truthiness with `if domain:`, and `None` and `""` are both falsy.

Reply:
Sorry, not sure, I didn't introduce that regex pattern - all I did was change the variable name